docs/en/ufs/S3.md

```diff
@@ -23,26 +23,22 @@ should also note the directory you want to use in that bucket, either by creatin
 the bucket, or using an existing one. For the purposes of this guide, the S3 bucket name is called
 `S3_BUCKET`, and the directory in that bucket is called `S3_DIRECTORY`.
 
-## Mounting S3
+## Basic Setup
 
 Alluxio unifies access to different storage systems through the
 [unified namespace]({{ '/en/advanced/Namespace-Management.html' | relativize_url }}) feature.
-An S3 location can be either mounted at
-the root of the Alluxio namespace or at a nested directory.
+An S3 location can be either mounted at the root of the Alluxio namespace or at a nested directory.
 
-### Root Mount
+### Root Mount Point
```

Comment:
for the part of describing different ways to set credential after "Specify the AWS credentials for S3 access.", can we move them to the `Advanced Setup` session, but using setting `aws.accessKeyId` and `aws.secretKey` in site properties in this `Basic Setup` session?

Reply:
Updated, thanks.